core/src/main/java/com/ibm/wala/ipa/callgraph/AnalysisScope.java

```diff
@@ -415,6 +415,7 @@ public ArrayClassLoader getArrayClassLoader() {
   /**
    * @return the rt.jar (1.4), core.jar (1.5), java.core.jmod (13) file, or null if not found.
    */
+  @SuppressWarnings("resource")
```

Comment:
Someday I'd really like to get [the Resource Leak Checker](https://checkerframework.org/manual/#resource-leak-checker) running on WALA to see how well it works...